pkg/controller/operators/olm/operator.go

```diff
@@ -1093,6 +1112,10 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 		}
 	case v1alpha1.CSVPhaseDeleting:
 		var immediate int64 = 0
+
+		if err := a.csvQueueSet.Remove(out.GetNamespace(), out.GetName()); err != nil {
```

Comment:
The name and namespace parameters are reversed here.

Reply:
Doesn't the following delete step result in the key being enqueued again?

Reply:
That explains some logs I saw :) If this is the only thing I will follow up with that fix.

Reply:
I have also fixed in #841